config/config.go

```diff
@@ -267,6 +267,86 @@ func init() {
 	}
 }
 
+// SaveConfigurableConsensus saves the configurable protocols file to the provided data directory.
+func SaveConfigurableConsensus(dataDirectory string, params ConsensusProtocols) error {
+	consensusProtocolPath := filepath.Join(dataDirectory, ConfigurableConsensusProtocolsFilename)
+
+	encodedConsensusParams, err := json.Marshal(params)
+	if err != nil {
+		return err
+	}
+	err = ioutil.WriteFile(consensusProtocolPath, encodedConsensusParams, 0644)
+	return err
+}
+
+// LoadConfigurableConsensusProtocols loads the configurable protocols from the data directroy
```

Comment:
It will be nice to add more details here: 
LoadConfigurableConsensusProtocols loads the configurable protocols and merges them with Consensus